install.sh

```diff
@@ -88,15 +93,15 @@ if [ -n "$HOSTNAME_PARAM" ] ; then
 	echo "-h $HOSTNAME_PARAM" >> "$HOST$DATADIR"/docker-run-opts
 	echo "$HOSTNAME_PARAM" > "$HOST$DATADIR"/hostname
 	OPTS="-h $HOSTNAME_PARAM"
-elif [ -z "$NET_HOST_PARAM" ] ; then
+else
```

Comment:
I'm not comfortable with the removal of that `elif [ -z "$NET_HOST_PARAM" ] ; then` without replacement. When neither `--net` nor `--hostname` are provided, won't things fail?

Reply:
Yes you're right - I've updated this snippet in my branch. Maybe the error message could be improved here to list all available options for setting the hostname.